products/caasp/main.pm

```diff
@@ -43,7 +43,7 @@ if (check_var('VIRSH_VMM_FAMILY', 'xen') && check_var('VIRSH_VMM_TYPE', 'linux')
     set_var('SERIALDEV', 'hvc0');
 }
 
-sub load_boot_tests {
+sub load_kubic_boot_tests {
```

Comment:
@kravciak are you fine with this rename to ensure we can use "load_boot_tests" to use that for the "common" ones or would you recommend to also rename the openSUSE/SLE ones?

Reply:
If rename helps you then I don't care, but I don't see the need for it - function is overloaded in caasp/main.pm and it does not conflict with SLE/main_common - or did I miss something?

Reply:
Well, I don't know how the overloading should work but perl reports an error (or at least a warning?) about the method in products/caasp/main.pm this is why I changed it.

Reply:
I see, then feel free to change it, but please don't use kubic naming scheme - it's a bit misleading in caasp/main.pm. What about:
```
load_boot_tests -> caasp_boot_tests
load_inst_tests -> load_oci_tests || caasp_inst_tests
```